p2p/host/hostv2/hostv2.go

```diff
@@ -108,13 +109,21 @@ func (host *HostV2) BindHandlerAndServe(handler p2p.StreamHandler) {
 
 // SendMessage a p2p message sending function with signature compatible to p2pv1.
 func (host *HostV2) SendMessage(p p2p.Peer, message []byte) error {
+	logger := log.New("from", host.self, "to", p, "PeerID", p.PeerID)
```

Comment:
let's use utils.GetLogInstance() for all the logging here - it does not make sense to create just one object for SendMessage.

Reply:
This child logger was used so that it adds the common parameters (from/to/PeerID) to all log messages emitted from this function.  However, I should've created it as a child of utils.GetLogInstance() and not log.Root().  Will keep in mind next time :-)